src/main/java/net/sf/jabref/JabRef.java

```diff
@@ -898,11 +889,11 @@ private static ParserResult importFile(String argument) {
      * @param argument See importFile.
      * @return ParserResult with setToOpenTab(true)
      */
-    private static ParserResult importToOpenBase(String argument) {
-        ParserResult result = JabRef.importFile(argument);
+    private static Optional<ParserResult> importToOpenBase(String argument) {
+        Optional<ParserResult> result = JabRef.importFile(argument);
 
-        if (result != null) {
-            result.setToOpenTab(true);
+        if (result.isPresent()) {
+            result.get().setToOpenTab(true);
         }
```

Comment:
Yes, there are a few more cases like that. I'll change them (now that I learned more about how to use Optional, didn't want to update during possible review...).